code/__DEFINES/jobs.dm

```diff
@@ -208,7 +181,16 @@
 #define JOB_DISPLAY_ORDER_DETECTIVE 36
 #define JOB_DISPLAY_ORDER_SECURITY_OFFICER 37
 #define JOB_DISPLAY_ORDER_PRISONER 38
->>>>>>> 90b974071de (Sign up for Cargorilla from the lobby (#79776))
+#define JOB_DISPLAY_ORDER_SECURITY_MEDIC 39 //SKYRAT EDIT ADDITON
+#define JOB_DISPLAY_ORDER_CORRECTIONS_OFFICER 40 //SKYRAT EDIT ADDITON
+#define JOB_DISPLAY_ORDER_NANOTRASEN_CONSULTANT 41 //SKYRAT EDIT ADDITON
+#define JOB_DISPLAY_ORDER_BLUESHIELD 42 //SKYRAT EDIT ADDITON
+#define JOB_DISPLAY_ORDER_ORDERLY 43 //SKYRAT EDIT ADDITION
+#define JOB_DISPLAY_ORDER_SCIENCE_GUARD 44 //SKYRAT EDIT ADDITION
+#define JOB_DISPLAY_ORDER_BOUNCER 45 //SKYRAT EDIT ADDITION
+#define JOB_DISPLAY_ORDER_ENGINEER_GUARD 46 //SKYRAT EDIT ADDITION
+#define JOB_DISPLAY_ORDER_CUSTOMS_AGENT 47 //SKYRAT EDIT ADDITION
+#define JOB_DISPLAY_ORDER_EXP_CORPS 48 //SKYRAT EDIT ADDITON
```

Comment:
why dont you guys just like move these to like 101, 102, 103 etc and never have to deal with it again